### ISSUE TITLE
pipeline options not being updated properly to fit_pipeline

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#184]
Could you please check if pipeline_config is being updated properly?
According to my debugging, it was not.

https://github.com/automl/Auto-PyTorch/blob/074a8f466b245e3c3852f28bc777783a8fcd6a1d/autoPyTorch/api/base_task.py#L1265

**Comment by ArlindKadra** [comment]
I think you have to do 
`from copy import copy`

or you could just do `=`
Although, I would suggest `deepcopy` from the context, since you are doing a weak copy with copy only.

**Comment by ravinkohli** [comment]
hey Arlind, thanks for pointing this bug. Actually, @franchuterivera has made a new PR #172 where he has fixed this. you can find the fix [here](https://github.com/automl/Auto-PyTorch/blob/c0ae704224256a9e316ab4f386c9776bcd3eefc6/autoPyTorch/evaluation/tae.py#L150). maybe we can rebase the cocktails branch from the refactor_development branch to also have the same implementation once this PR is merged

**Comment by ravinkohli** [comment]
As the corresponding PR #172 has been merged, I'm closing this issue. If there's still something missing, feel free to reopen
